Create log-master

ImTedious opened the pull request:
Added file in plugins for log-master

Log-master is a plugin that adds a task list in a dashboard on the collection log interface which I will use to further my youtube series (Tedious' GenerateTask).

Nightfirecat commented:
[Don't use "example" as your config group](https://github.com/ImTedious/generate-task/blob/de5d07de19b298c4f642ba1ba11c3444803336aa/src/main/java/com/logmaster/LogMasterConfig.java#L7). In fact, you could delete your config interface entirely as it's not used, if you like.
LGTM otherwise

ImTedious commented:
I have changed "example" to "log-master" and updated the commit hash.
Do I need to do a new pull request or will you be able to push it through
now?
Thanks!

Kind regards

Op do 29 jun 2023 om 03:57 schreef Jordan ***@***.***>:

> Don't use "example" as your config group
> <https://github.com/ImTedious/generate-task/blob/de5d07de19b298c4f642ba1ba11c3444803336aa/src/main/java/com/logmaster/LogMasterConfig.java#L7>.
> In fact, you could delete your config interface entirely as it's not used,
> if you like.
> LGTM otherwise
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/runelite/plugin-hub/pull/4517#issuecomment-1612320411>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5IBKOMMXGYQ5JQXUOCPMQ3XNTOHTANCNFSM6AAAAAAZPHZW4E>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
